### ISSUE TITLE
How to intergate my own chabot to my website

### ISSUE THREAD

**Issue by shubhamnagalwade** [Abdulkereem/flask-machine-learning-chatbot#4]
Do you have any idea for that how t integrated?

**Comment by Abdulkereem** [comment]
yes i can help with that, kindly email me abdulkereemokereem@gmail.com

**Comment by shubhamnagalwade** [comment]
> abdulkereemokereem@gmail.com

thanks

**Comment by shubhamnagalwade** [comment]
thanks

Thanks and Regards,
Shubham Nagalwade
Data Scientist.

Contact : http://www.linkedin.com/in/shubham-nagalwade-1b99b6132 | +91
9765202116
